Automattic/wp-calypso
# Refine Jetpack Sidebar

#### Changes proposed in this Pull Request

This is a catch-all for some Jetpack Cloud Sidebar improvements:

* Use selectedSite to improve sidebar links
* Remove sub items in line with design
* Improve selection logic
* Add Settings icon

<img width="1222" alt="Screen Shot 2020-02-27 at 1 56 03 PM" src="https://user-images.githubusercontent.com/2810519/75490380-f891d880-5968-11ea-94d6-8bf60221caf3.png">

#### Testing instructions



1. Navigate to the root of Jetpack Cloud
2. Verify "Dashboard" is selected and the current site indicator is hidden
3. Navigate to "Backups" and select a site
4. Verify that the remaining links to `Scan` and `Settings` now use the selected site slug
5. Verify that "Backups" is selected and the current site indicator is shown
6. Repeat 3 -5 for "Scan" and "Settings"
